pkg/providers/aws/provider_postgres_test.go

```diff
@@ -654,6 +654,7 @@ func buildAvailableDBInstance(testID string) []*rds.DBInstance {
 			DBInstanceStatus:           aws.String("available"),
 			AutoMinorVersionUpgrade:    aws.Bool(false),
 			AvailabilityZone:           aws.String("test-availabilityZone"),
+			CACertificateIdentifier:    aws.String(newCert),
```

Comment:
seems like a different PR, can we try resolving pulling these?

Reply:
Thank you for comment. Rebase done, but it was complex, and this remains, But changes from other PRs were added as separate commit's, and it can be found in the list of commit's. Yes, author at this line appears wrong, but maybe it will be changed after merge (not sure). 
It can happens, and difficult to avoid, just rework, but PR is too big for this. So, I'm sorry that Author at this line can appear not correctly.

Reply:
Done. removed/reverted - https://github.com/integr8ly/cloud-resource-operator/commit/bd3644333a0770a7ca360e0e049ba3000bf714bc